SimplenoteWidgets/Views/ListWidgetHeaderView.swift

```diff
@@ -0,0 +1,41 @@
+import SwiftUI
+import WidgetKit
+
+
+struct ListWidgetHeaderView: View {
+    let tag: WidgetTag
+
+    @Environment(\.widgetFamily) var widgetFamily
+
+    var body: some View {
+        HStack(alignment: .center) {
+            Link(destination: URL.internalUrl(forTag: tag.identifier)) {
+                Text(displayName(for: tag.tagDescription))
+                    .font(.headline)
+                    .foregroundColor(.bodyTextColor)
+                Spacer()
+            }
+            Link(destination: URL.newNoteURL(withTag: tag.identifier)) {
+                NewNoteImage(size: Constants.side,
+                             foregroundColor: .widgetTintColor,
+                             backgroundColor: .widgetBackgroundColor)
+            }
+        }
+        .padding(.zero)
+    }
+}
+
+private func displayName(for tag: String) -> String {
+    tag.replacingOccurrences(of: "-", with: " ")
+}
```

Comment:
I had found it more readable, but you are correct we do display the tag as is everywhere else.  I can drop this WDYT?

Reply:
@SylvesterWilmott, should we do anything with a tag when we display it as a title of the widget or just show it as it is?

As of now, when tag is `hello-world` it will be shown in the note as `hello-world` and in the widget as `hello world` (notice that `-` is replaced with space).

Reply:
Ah, I did not notice this before! We should definitely display the tag as is character for character.